resources/js/common.js

```diff
@@ -283,3 +282,40 @@ function postwith (to,p) {
   document.body.removeChild(myForm) ;
 }
 
+
+function isValidDate(dateString)

```

Comment:
@remocrevo 
> Note: CORAL currently uses jQuery 1.4.4.

Actually Coral uses [4 different versions of jQuery](https://github.com/Coral-erm/Coral/issues/84#issuecomment-236202203) so we have a broad choice ! ^^"